README.md

```diff
@@ -71,7 +71,7 @@ up a "local remote" on your file system:
 > the DVC project.
 
 ```console
-$ mkdir -P /tmp/dvc-storage
+$ mkdir -p /tmp/dvc-storage
```

Comment:
`-p` was OK I think

https://www.gnu.org/software/coreutils/manual/html_node/mkdir-invocation.html

Reply:
Oh I see you changed it to `-p`. Never mind